gitlint/rules.py

```diff
@@ -236,6 +236,20 @@ def validate(self, title, _commit):
             return [RuleViolation(self.id, violation_msg, title)]
 
 
+class TitleMinLength(LineRule):
+    name = "title-min-length"
+    id = "T8"
+    target = CommitMessageTitle
+    options_spec = [IntOption('min-length', 5, "Min line length in characters")]
+
+    def validate(self, title, _commit):
+        min_length = self.options['min-length'].value
+        actual_length = len(title)
+        if 0 < actual_length < min_length:
```

Comment:
This would still allow for empty commit titles. I'd expect we'd want to capture this too?
```suggestion
        if actual_length < min_length:
```

I think it would be good to add a test for an empty title specifically to cover this scenario